apps/web/hooks/use-agent.ts

```
@@ -0,0 +1,240 @@
+    createThreadItem(aiThreadItem);
+    setCurrentThreadItem(aiThreadItem);
+    setIsGenerating(true);
+    setCurrentSources([]);
+
+    console.log("formData", formData)
+
+    startWorkflow({
+      mode: chatMode as any,
+      question: formData.get('query') as string,
+      threadId,
+      messages: [...(threadItems?.flatMap(item => [{
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)